lib/jormungandr/src/Cardano/Wallet/Jormungandr/Network.hs

```diff
@@ -168,9 +173,9 @@ data JormungandrLayer m = JormungandrLayer
     , postMessage
         :: (Tx, [TxWitness])
         -> ExceptT ErrPostTx m ()
-    , getInitialFeePolicy
+    , getInitialConfigParams
         :: Hash "Genesis"
-        -> ExceptT ErrGetInitialFeePolicy m FeePolicy
+        -> ExceptT ErrGetInitialConfigParams m (FeePolicy, SlotLength)
```

Comment:
As mentioned in comment (and as you suggested I believe?), the return type could really be: `BlockchainParameters`

Reply:
done